Frontend/src/app.tsx

```diff
@@ -13,7 +13,8 @@ import ProgressBar from 'src/components/progress-bar';
 import { MotionLazy } from 'src/components/animate/motion-lazy';
 import { SettingsDrawer, SettingsProvider } from 'src/components/settings';
 
-import { AuthProvider } from 'src/auth/context/jwt';
+// import { AuthProvider } from 'src/auth/context/jwt';
```

Comment:
Lets remove all comments. IF you want them back, you can open github and check this commit to see which lines have been removed.